specs/agents/metrics-otel.md

```diff
@@ -0,0 +1,181 @@
+
+Agents MUST use the OpenTelemetry SDK [default aggregations per instrument type](https://opentelemetry.io/docs/reference/specification/metrics/sdk/#default-aggregation) as default aggregations for their exporter:
+
+| **Instrument Type**         | **Aggregation**            |
+|-----------------------------|----------------------------|
+|  Counter                    | Sum                        |
+|  Asynchronous Counter       | Sum                        |
+|  UpDownCounter              | Sum                        |
+|  Asynchronous UpDownCounter | Sum                        |
+|  Asynchronous Gauge         | Last Value                 |
+|  Histogram                  | Explicit Bucket Histogram  |
+
+### Histogram Aggregation
+
+The OpenTelemetry SDK uses the following prometheus-inspired bucket boundaries for histograms by default:
+```
+[0, 5, 10, 25, 50, 75, 100, 250, 500, 750, 1000, 2500, 5000, 7500, 10000]
+```
+
+Because IntakeV2 histogram serialization skips empty buckets, we are free to use histogram boundaries which are more accurate and cover a greater range. Therefore, agents SHOULD use the following histogram boundaries by default instead:
+
+```
+[0.00390625, 0.00552427, 0.0078125, 0.0110485, 0.015625, 0.0220971, 0.03125, 0.0441942, 0.0625, 0.0883883, 0.125, 0.176777, 0.25, 0.353553, 0.5, 0.707107, 1, 1.41421, 2, 2.82843, 4, 5.65685, 8, 11.3137, 16, 22.6274, 32, 45.2548, 64, 90.5097, 128, 181.019, 256, 362.039, 512, 724.077, 1024, 1448.15, 2048, 2896.31, 4096, 5792.62, 8192, 11585.2, 16384, 23170.5, 32768, 46341.0, 65536, 92681.9, 131072]
+```
+
+These boundaries are an exponential scale base `sqrt(2)` rounded to six significant figures.
+
+Agents MUST allow users to configure the default histogram boundaries via the `custom_metrics_histogram_boundaries` configuration option:
+
+|                |                                          |
+|----------------|------------------------------------------|
+| Type           | `double list`                            |
+| Default        | <see base `sqrt(2)` boundaries above>    |
+| Dynamic        | `false`                                  |
+| Central config | `false`                                  |
+
+In the future, we might replace the default aggregation with one better suited for elasticsearch, e.g. HDRHistogram or T-Digest.
+
+Because users would need to take extra steps to prevent our exporter from using the preferred explicit bucket aggregation (by configuring SDK views), agents are not required to support any other histogram aggregation types. In other words, agents MAY ignore [ExponentialHistograms](https://opentelemetry.io/docs/reference/specification/metrics/data-model/#exponentialhistogram) and [Summaries](https://opentelemetry.io/docs/reference/specification/metrics/data-model/#summary-legacy). If a metric is ignored due to its aggregation, agents SHOULD emit a warning once with a message including the metric name.
```

Comment:
```suggestion
Because users would need to take extra steps to prevent our exporter from using the preferred explicit bucket aggregation (by configuring SDK views), agents are not required to support any other histogram aggregation types. In other words, agents MAY ignore [ExponentialHistograms](https://opentelemetry.io/docs/reference/specification/metrics/data-model/#exponentialhistogram) and [Summaries](https://opentelemetry.io/docs/reference/specification/metrics/data-model/#summary-legacy). If a metric is ignored due to its aggregation, agents SHOULD emit a warning with a message including the metric name.
```